resources/default_pipeline_environment.yml

```diff
@@ -6,6 +6,7 @@
 #Project Setup
 general:
   collectTelemetryData: true
+  logFormat: 'plain'
```

Comment:
looks like this changes the default ...
Do we really want this?

I would rather stick with the more complex format since it provides additional information which can be important in error cases ...

Reply:
Honestly, I think the default log format makes the log harder to read which makes the jenkins pipeline that uses the go binary worse to use compared to before, cf screenshot. 

![image](https://user-images.githubusercontent.com/2292245/80484136-82552680-8957-11ea-98ee-cc8e7aa12c0d.png)

Maybe we can find a better compromise, I see your points and they are valid, but I think readability of log output is also very important.

Reply:
yes, we might bring in s.th in between with **a fixed width layout**, e.g.

<timestamp> <Level 1 char> <Library (OS,SAP, ...)> <stepName> - <message>

Based on the learnings we need following info as quickly as possible:
- timestamp (-> we probably should not rely on the CI system here)
- which libarary is used -> if empty we know it is non-productive or custom ...
- Error category
- stepName, especially important when parallel executions happen!

could look like:

```
08:52:20 I OS nodeJsBuild - Received 782K...
08:52:20 I OS nodeJsBuild - Received 1668K...
...
08:52:21 I OS nodeJsBuild - Copying to target path ...
```
Not sure if a 1 char category is too good, maybe rather 4 chars (INFO, WARN):

```
08:52:20 INFO OS nodeJsBuild - Received 782K...
08:52:20 INFO OS nodeJsBuild - Received 1668K...
...
08:52:21 INFO OS nodeJsBuild - Copying to target path ...
```